web/src/lib/components/photos-page/actions/delete-assets.svelte

```diff
@@ -34,7 +34,7 @@
 
   const handleDelete = async () => {
     loading = true;
-    const ids = [...getOwnedAssets()].filter((a) => !a.isExternal).map((a) => a.id);
+    const ids = [...getOwnedAssets()].map((a) => a.id);
```

Comment:
this might lead to weird behavior as it will actually delete assets that are from read only external libraries and then they will get added back on the next scan, but it's fine with me.

Reply:
updated to check that they're not readonly